sensor/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+// HasAPI checks whether the kubernetes client supports the groupVersion API for the specified kind
```

Comment:
Spotted just now
```suggestion
// HasAPI checks whether the kubernetes server supports the groupVersion API for the specified kind
```

Reply:
Fixed, I had just moved this function as is from elsewhere